File: src/main/java/io/stouder/adonis/edge/parsing/EdgeElementType.java
package io.stouder.adonis.edge.parsing;

import com.intellij.psi.tree.IElementType;
import io.stouder.adonis.edge.EdgeLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class EdgeElementType extends IElementType {
    public EdgeElementType(@NonNls @NotNull String debugName) {
        super(debugName, EdgeLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "[Edge] " + super.toString();
    }
}

File: src/main/java/io/stouder/adonis/edge/parsing/EdgeElementType.kt
package io.stouder.adonis.edge.parsing

import com.intellij.psi.tree.IElementType
import io.stouder.adonis.edge.EdgeLanguage

class EdgeElementType(debugName: String) : IElementType(debugName, EdgeLanguage.INSTANCE) {
    override fun toString(): String {
        return "[Edge] ${super.toString()}"
    }
}
